src/translations/enTranslations.json

```
@@ -1368,7 +1368,8 @@
     "sort_by": "Order by",
     "sort_by_modified_date": "Last modified",
     "sort_by_published_date_asc": "Publish date (oldest first)",
-    "sort_by_published_date_desc": "Publish date (newest first)"
+    "sort_by_published_date_desc": "Publish date (newest first)",
```

Litt usikker på oversettelse her.

```suggestion
    "sort_by_published_date_desc": "Publication date (newest first)",
```

---

Ja, litt tvetydig den der... Publication brukes også som begrep noen plasser, men `Published date` er nok en bedring fra dagens!